cmd/create/cluster/cmd_test.go

```diff
@@ -136,3 +138,39 @@ var _ = Describe("Validates OCP version", func() {
 		})
 	})
 })
+
+var _ = Describe("Validate cloud accounts", func() {
+
+	Context("build billing accounts", func() {
+		When("return the error or result", func() {
+			It("OK: Successfully gets contracts from cloudAccounts", func() {
+				mockCloudAccount := v1.NewCloudAccount().CloudAccountID("1234567").
+					Contracts(v1.NewContract().StartDate(time.Now()).EndDate(time.Now().Add(2)).
+						Dimensions(v1.NewContractDimension().Name("control_plane").Value("4")))
+				cloudAccount, err := mockCloudAccount.Build()
+				Expect(err).NotTo(HaveOccurred())
+				_, isContractEnabled := GetBillingAccountContracts([]*v1.CloudAccount{cloudAccount}, "1234567")
+				Expect(isContractEnabled).To(Equal(true))
+			})
+
+			It("OK: Successfully print contract details", func() {
+				mockContract, err := v1.NewContract().StartDate(time.Date(2023, 10, 12, 12, 12, 12, 12, time.UTC)).
+					EndDate(time.Date(2023, 10, 12, 12, 12, 12, 12, time.UTC)).
+					Dimensions(v1.NewContractDimension().Name("control_plane").Value("4")).Build()
```

Comment:
Let's test the `four_vcpu_hour` dimension too

Reply:
Added the test.